src/app/healthcheck/middleware.js

```diff
@@ -0,0 +1,10 @@
+const axios = require("axios");
```

Comment:
Do you need this require?

Reply:
Thanks mate, nope  (I caught it before but didn't actually add it to the commit 🤦 , updated and pushed)